docs/docs/10.8-pure-render-mixin.md

```diff
@@ -6,6 +6,8 @@ prev: update.html
 next: perf.html
 ---
 
+> Note: React 15.3 added the [React.PureComponent](https://github.com/facebook/react/releases/tag/v15.3.0) base class. This is a more convenient way to implement the behavior below that does not rely upon mixins. React.PureComponent is the recommend approach when working with ES2015 classes.
```

Comment:
We could potentially make the note a little more aggressive by replacing the entire note with something like... `PureRenderMixin` (for createClass components) has been replaced by `React.PureComponent` (for ES6 components) in React 15.3.  The `React.PureComponent` base class is now preferred over `PureRenderMixin`.